site/package.json

```diff
@@ -11,7 +11,8 @@
   "scripts": {
     "start": "npm run sync && run-p start:*",
     "build": "npm run clean && run-s build:*",
-    "build:docs": "node ../scripts/docs.js",
+    "postbuild": "cp src/_redirects dist/_redirects",
```

Comment:
Does this need to happen after the build? Or could it be a `build:redirects` step?  If so, lets just make it a build step  so that insinuation isn't there.  If it needs to be post build, then fine.